docs/UserGuide.md

```diff
@@ -60,7 +61,7 @@ statement_guid,property_id,wikidata_value,meta_wikidata_value,external_value,ext
 
 _**Note**: The data `wikidata_value`, `external_value`, `external_url` should be limited to a length of 1500 characters maximum._
 
-Additionally, each row of the csv file must contain exactly 4 commas (`,`). Optional values can simply be left out.
+All columns must be present; optional values can be left empty.
```

Comment:
Mentioning that leaving empty is putting commas is worth it here. I am not sure if all users will be familiar with how empty fields work in csv. :thinking:

Reply:
Maybe, but I’m not sure how to phrase it… I guess we can use a specific column as an example?

> , e.g. `,,` for empty `meta_wikidata_value`.

Reply:
Done.